src/components/mapComponents/logic/event.ts

```diff
@@ -158,28 +158,24 @@ export function addHandleZoomChange(
         setBlocksStyle(blocksLayer, zoomedIn);
         break;
       case MapViews.TREES:
-        setSitesStyle(
-          sitesLayer,
-          visibleSites,
-          getImageSize(zoomLevel, view),
-          zoomedIn,
-        );
+        setSitesStyle(sitesLayer, visibleSites, zoomLevel, zoomedIn);
         break;
     }
   });
 }
 
 /**
  * Determines the image size given the zoom level and the view.
- * @param zoomLevel the zoom level
- * @param view the view
+ * @param zoomLevel the zoom level between 16 and 22 where zooming in increases zoom level
  */
-export function getImageSize(zoomLevel: number, view: MapViews): number {
-  let imageSize = 0;
-  if (zoomLevel >= STREET_ZOOM) {
-    imageSize = 2;
-  } else if (zoomLevel > view + 1) {
-    imageSize = 1;
+export function getImageSize(zoomLevel: number): number {
+  if (zoomLevel >= 21) {
+    return 35;
+  } else if (zoomLevel >= 19) {
+    return 20;
+  } else if (zoomLevel >= 18) {
+    return 13;
+  } else {
+    return 5;
```

Comment:
Just curious, how would scaling the icon size as a function of the zoom level look compared to discrete size intervals?

Reply:
That's an excellent question and it's actually what we did at first. But the function was quadratic (linear wasn't good enough sadly) with seemingly random constants (`0.23 * Math.pow(zoomLevel, 2) - 3.94 * zoomLevel + 12.21` to be exact). Which do you think is better?

Reply:
I like the idea of a function as well, maybe we could do a regression to find one that fits our existing discrete points?

Reply:
This is the function we had: `0.23 * Math.pow(zoomLevel, 2) - 3.94 * zoomLevel + 12.21`. We might be able to round the constants to the nearest tenths, do you think this would be better?

Reply:
Fitting our points to a regression could make changing the sizes in the future a lot harder, since whoever's making the change may not know which points it's fit to and will have to play around with the coefficients to make something work. If we want to do a function, would it be possible to save it until the very end for our sake?